libs/ui/test/ui_strings/test_strings.tsx

```diff
@@ -0,0 +1,64 @@
+/* istanbul ignore file - test utils */
+
+import { LanguageCode, UiStringsPackage } from '@votingworks/types';
+
+import React from 'react';
+import { UiString } from '../../src/ui_strings/ui_string';
+import { Font } from '../../src/typography';
+
+export const TEST_UI_STRING_KEY = 'numPlanets';
+
+export const TEST_UI_STRING_TRANSLATIONS: UiStringsPackage = {
+  [LanguageCode.ENGLISH]: {
+    [`numPlanets_one`]: 'There is only <2>{{count}}</2> planet.',
+    [`numPlanets_other`]: 'There are <2>{{count}}</2> planets.',
+    planetName: {
+      planet1: 'Mercury',
+      planet3: 'Earth',
+      planet9: 'Pluto', // #NeverForget
```

Comment:
Was going to comment on the boldness of your decision to still include Pluto after seeing it in the PR description 😆